lightkurve/targetpixelfile.py

```diff
@@ -858,7 +917,7 @@ def mission(self):
         except KeyError:
             return None
 
-    def aperture_photometry(self, aperture_mask='pipeline'):
+    def aperture_photometry(self, aperture_mask='pipeline', **kwargs):
```

Comment:
Are we happy that we're now passing strings ('pipeline', 'threshold') or boolean arrays to the same keyword?